### DISCUSSION TITLE
Mensaje de error de la ejecución de ejemplos/tutoriales

### DISCUSSION THREAD

**Discussion by RJacque** [IDAES/idaes-pse#1340] [General]
Hola buen día,

Actualmente acabo de instalar WaterTAP3 y en el proceso de prueba de la instalación del software me aparece este error:

ModuleNotFoundError                       Traceback (most recent call last)
Cell In[9], line 1
----> 1 from watertap3.utils import watertap_setup, get_case_study, run_model, run_watertap3
      2 import pandas as pd
      3 import numpy as np

File c:\users\usuario.t-3246\desktop\water tap 3\watertap3\watertap3\watertap3\utils\__init__.py:6
      4 from . import cost_curves
      5 from .cost_curves import *
----> 6 from . import design
      7 from .design import *
      8 from . import financials

File c:\users\usuario.t-3246\desktop\water tap 3\watertap3\watertap3\watertap3\utils\design.py:3
      1 from . import module_import
      2 from .constituent_removal_water_recovery import create
----> 3 from .mixer_wt3 import Mixer
      4 from .source_wt3 import Source
      6 __all__ = ['add_unit_process',
      7            'add_water_source',
      8            ]

File c:\users\usuario.t-3246\desktop\water tap 3\watertap3\watertap3\watertap3\utils\mixer_wt3.py:19
     17 from pyomo.environ import (Reals, SolverFactory, TerminationCondition, Var)
     18 from pyomo.common.config import ConfigBlock, ConfigValue, In
---> 19 from idaes.core import (UnitModelBlockData, declare_process_block_class, useDefault)
     20 from idaes.core.util.config import (is_physical_parameter_block, is_state_block, list_of_strings)
     21 from idaes.core.util.exceptions import (BurntToast, ConfigurationError, PropertyNotSupportedError)

ModuleNotFoundError: No module named 'idaes'

¿Cómo puedo corregirlo?

**Comment by andrewlee94** [comment]
Hello @RJacque - from an initial look at the error, it would appear that `idaes` is not installed in your environment for some reason. What were the steps you used to install WaterTAP?

Notably, if you did a developer install (https://watertap.readthedocs.io/en/stable/getting_started.html#for-watertap-developers), make sure you ran the following command correctly: `pip install -r requirements-dev.txt`.

**Comment by lbianchi-lbl** [comment]
He notado que el `watertap` en esos mensajes de error parece ser diferente de "nuestro" WaterTAP, es decir, https://github.com/watertap-org/watertap. ¿Puedes proporcionar más detalles sobre cómo instalaste ese `water tap 3` código?

En cualquier caso, para resolver el `ModuleNotFoundError`, puedes intentar ejecutar `pip install idaes-pse` en el mismo entorno donde estás ejecutando el código.

**Reply by RJacque** [reply]
Hola, buen día. He Instalado la versión que me recomendó, pero no puedo cargar las unidades de proceso individuales como puedo usar esta versión para crear un diseño propio de una planta de tratamiento y correrlo en este programa WaterTAP UI?

**Reply by andrewlee94** [reply]
Hello again, at this point this is sounding more like a question for the WaterTAP team who have their own GitHub page for discussion: https://github.com/orgs/watertap-org/discussions

The IDAES team does not have any involvement in the WaterTAP UI, so unfortunately we cannot give any guidance on how to resolve this.

**Reply by RJacque** [reply]
Muchas gracias por tu respuesta.

**Comment by RJacque** [comment]
Hola Ya se instaló el recomendado y ya está funcionando bien muchas gracias.
